docs/workforce-management-framework/topics/con-introducing-securityfeatures.adoc

```diff
@@ -22,15 +22,15 @@ NOTE: All link:https://expressjs.com/[express] routes outside of {PROJECT_NAME}
 == Introducing Passport.js
 
 *Passport.js* is a lightweight authentication middleware solution for Node.js and can be bundled into an existing Node.js server.
-*Passport.js* offers a comprehensive set of link:http://passportjs.org/docs/configure[strategies] that support authentication using a username and password, Facebook logon, Twitter logon, and more.
+*Passport.js* offers a comprehensive set of link:http://passportjs.org/docs/configure[strategies] that support authentication using a username and password, existing social media and other 3rd party accounts, and more.
```

Comment:
Here I'm not sure if it's okay for us to include references to these services by name